### DISCUSSION TITLE
Hybrid battery support

### DISCUSSION THREAD

**Discussion by voodoo** [jenavarro/obdlogger#34] [Ideas]
It would be neat to have readings from the hybrid battery in a Prius (or other hybrid). I could not see the 'PID' for those sensors. 

Also, if it uses native browser/web Bluetooth rather than having to go through cordova. Though it seems like that might primarily be for advertising. Yikes. 👎

https://googlechrome.github.io/samples/web-bluetooth/

**Comment by jenavarro** [comment]
That is interesting. Don't know if those cars support additional PIDs..
check this out: https://hybridassistant.blogspot.com/p/about.html
Regarding cordova vs native browser.... it is a possibility I have not
explored. For sure the key will be having support for networking detection,
preventing app from sleeping, sqlite-like database, and others that cordova
support and the project uses...

Tks.
Javier
